From 402a2c8ed1e509a917cf7a22609e49aea2bc0921 Mon Sep 17 00:00:00 2001
From: wanglimin <wanglimin@xfusion.com>
Date: Wed, 21 Dec 2022 09:49:14 +0800
Subject: [PATCH] revert any to interface{} temporarily to allow builtable with
 golang-1.17.x it will be withdrawed if golang upgrade to 1.18.x in the branch

---
 vendor/archive/tar/common.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vendor/archive/tar/common.go b/vendor/archive/tar/common.go
index c99b5c1..595de64 100644
--- a/vendor/archive/tar/common.go
+++ b/vendor/archive/tar/common.go
@@ -538,7 +538,7 @@ type headerFileInfo struct {
 func (fi headerFileInfo) Size() int64        { return fi.h.Size }
 func (fi headerFileInfo) IsDir() bool        { return fi.Mode().IsDir() }
 func (fi headerFileInfo) ModTime() time.Time { return fi.h.ModTime }
-func (fi headerFileInfo) Sys() any           { return fi.h }
+func (fi headerFileInfo) Sys() interface{}   { return fi.h }
 
 // Name returns the base name of the file.
 func (fi headerFileInfo) Name() string {
-- 
2.21.0

